src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # SARIF Package Release History (SDK, Driver, Converters, and Multitool)
 
+## Unreleased
+
+* BUGFIX: Fix issue `match-results-forward` command generates invalid Sarif when results contain sub ruleId. [#2486](https://github.com/microsoft/sarif-sdk/pull/2486)
```

Comment:
>BUGFIX: Fix issue `match-results-forward` command generates invalid Sarif when results contain sub ruleId. [#2486](https://github.com/microsoft/sarif-sdk/pull/2486) [](http://example.com/codeflow?start=2&length=164)

I'm not sure if this is just unclear or not descriptive enough. What was invalid about the SARIF results before this fix?

Reply:
The issue of invalid Sarif produced before the fix is it contains duplicated rule definitions in rules array if the Sarif results contain results have sub ruleId.

E.g. if there are 2 results' ruleId is "TESTRULE/001", the rules array creates 2 duplicated rules of "TESTRULE".

Will update the description with the duplicated rules issue, and update the issue with the detail errors. What do you think?

Reply:
So, this is fixing an issue where the `match-results-forward` command generates Sarif with duplicated results if there is a sub `ruleId`?
Yes, please update both.

Reply:
And just to be clear, also update the release note here. If the above captures everything, you can copy/paste most of it. Don't forget the backticks around ruleId.